accelerator/proxy_apis.h

```diff
@@ -13,6 +13,8 @@
 #include "cclient/api/core/core_api.h"
 #include "cclient/request/requests.h"
 #include "cclient/response/responses.h"
+#include "serializer/serializer.h"
+#include "utils/hash_algo_djb2.h"
```

Comment:
Why did you include `hash_algo_djb2.h`?

Reply:
Include it in C source rather than in header files.

Reply:
OK